src/lib.rs

```diff
@@ -93,7 +95,7 @@ use core::mem::ManuallyDrop;
 /// };
 /// assert_eq!(rebuilt, [4294967295, 0, 1]);
 /// ```
-#[derive(Debug, PartialEq, Eq, Hash)]
+#[derive(Eq)]
```

Comment:
I think clippy will complain when deriving `Eq` when `PartialEq` is implemented manually.

Please remove the `#[derive(Eq)]` and replace it with:

```rust
impl<T> Eq for RawParts<T>
```

Please put this below the `PartialEq` implementation.

Reply:
Will do.